### ISSUE TITLE
After Browser restart, Payments Flag wont toggle

### ISSUE THREAD

**Issue by mandar-brave** [brave/browser-laptop#13227]
### Description

See issue: https://github.com/brave/browser-laptop/issues/13226
After restarting the browser, all other issues (tabs, etc.) are fixed but unable to toggle the payments flag.

### Steps to Reproduce

  1. See https://github.com/brave/browser-laptop/issues/13226
  2. Restart browser
  3. Issue shows up

**Actual result:**

Should be able to toggle flags and use all functionality as expected
![image](https://user-images.githubusercontent.com/35932307/36493614-dca067cc-16e4-11e8-9517-beebbb424f50.png)

**Expected result:**

**Reproduces how often:**

### Brave Version

**about:brave info:**

v.20 (Latest production release on Feb 18, 2018)

**Reproducible on current live release:**

Yes

**Comment by bsclifton** [comment]
Should definitely be retested once we have a 0.21.x release candidate available.  Some broken functionality was fixed with https://github.com/brave/browser-laptop/pull/13234

**Comment by NejcZdovc** [comment]
@mandar-brave does this still happen to you after 0.21 release?

**Comment by mandar-brave** [comment]
Not yet @NejcZdovc - i upgraded this AM. would love to keep trying for another 2 days before we close.

**Comment by KarmaMonk** [comment]
Yes it still happens with 0.21.24.

After selecting the payment "menu" in Preferences everything in Preferences is blocked. 

Brave: 0.21.24 
V8: 6.5.254.36 
rev: 19c78a1624118e344852c4c0f44c0d45be841b5f 
Muon: 4.9.3 
OS Release: 4.14.0-041400-generic 
Update Channel: Release 
OS Architecture: x64 
OS Platform: Linux 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 65.0.3325.162

![error-brave-payments](https://user-images.githubusercontent.com/25179545/37905603-a8b1810a-30ff-11e8-8cc4-09eb6f13b464.png)

**Comment by NejcZdovc** [comment]
Let's check why button is disabled. We need to enable it in this case and retry wallet creation

**Comment by KarmaMonk** [comment]
In my case ...
- when I accessed the payment page directly via about:preferences#payments, the payment page broke down, 
- when I accessed the payment page over the settings>payments Menu it blocked the paymant page ant the whole settings Tab. This error was still in place with version 0.22.16 or 0.22.17 (not sure).

Some days ago (april 11th) I deleted the whole .config/brave Folder. After the restart I recovered my bat-account and Bookmarks.  Since then everything seams to work fine. Looking forward for the next payment at may 11.
